ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -10683,15 +10685,10 @@ ABoundActivity * HqlCppTranslator::doBuildActivityOutput(BuildCtx & ctx, IHqlExp
         StringBuffer s;
         s.append(getActivityText(kind));
         s.append("\n");
-        if (expr->hasAttribute(_spill_Atom))
-            s.append("Spill File");
-        else
-        {
-            if (expr->hasAttribute(_workflowPersist_Atom))
-                s.append("Persist ");
+        if (expr->hasAttribute(_workflowPersist_Atom))
+            s.append("Persist ");
```

Comment:
Should a label be generated for spillwrite ?

Reply:
no, it already says "Spill Write2  "Spill write Spill File" is too verbose.  It should probably remove the \n as well though - I will fix.

Reply:
Actually it is correct as it is - spilled writes will not go through this branch